mne/viz/tests/test_topomap.py

```diff
@@ -109,6 +110,13 @@ def test_plot_topomap():
                     for x in subplot.get_children()
                     if isinstance(x, matplotlib.text.Text)))
 
+    # Plot array
+    for ch_type in ('mag', 'grad'):
+        evoked_ = evoked.copy().pick_types(eeg=False, meg=ch_type)
+        plot_topomap(evoked_.data[:, 0], evoked_.info)
```

Comment:
Well I don't know but this PR doesn't change any of this. The PR doesn't touch anything beyond getting the pos array.